services/src/test-library/src/main/java/org/openkilda/testing/service/database/DatabaseSupportImpl.java

```diff
@@ -300,6 +305,18 @@ public Flow getFlow(String flowId) {
         return flowRepository.findById(flowId).get();
     }
 
+    /**
+     * Get flowPair.
+     *
+     * @param forwardPathId forward path Id
+     * @param reversePathId reverse path Id
+     * @return Collection  of TransitVlan
```

Comment:
afaik our checkstyle policy does not force you to write this kind of useless docs. If you have nothing to add - do not add. This kind of doc is just a distraction and waste of space. You'd better describe when it returns 1 vlan vs cases when it returns 2 vlans

Reply:
description was updated